docs/features/job-queue.txt

```diff
@@ -1,103 +1,170 @@
++---------------+-----------------------------------------------------------------------+
+| ``Scheduled`` | Scheduled to run or running, and resources might have been allocated. |
++---------------+-----------------------------------------------------------------------+
 
-Completed or errored jobs are not counted as active and are omitted from this list.
+Completed or errored jobs are not counted as active and are omitted from the queued job list.
 
-*************************
- Visualize the Job Queue
-*************************
+********************
+ View the Job Queue
+********************
 
 You can view the job queue using the CLI or WebUI. In the WebUI, click the **Job Queue** tab. In the
 CLI, use one of the following commands:
 
-.. code::
+.. code:: bash
 
    $ det job list
+
+or
+
+.. code:: bash
+
    $ det job ls
 
 These commands show the default resource pool queue. To view other resource pool queues, use the
 ``--resource-pool`` option, specifying the pool:
 
-.. code::
+.. code:: bash
 
    $ det job list --resource-pool compute-pool
 
 For more information about the CLI options, see the CLI documentation or use the ``det job list -h``
 command.
 
-The WebUI and the CLI display a table of results, ordered by scheduling order. The scheduling order
-does not represent the job priority. In addition to job order, the table includes the job states and
-number of slots allocated to each job.
+The WebUI and the CLI display a results table ordered by scheduling order. The scheduling order does
+not represent the job priority. In addition to job order, the listing shows:
+
++---------------+------------------------------------------------------------------------------+
+| Job Property  | Description                                                                  |
++===============+==============================================================================+
+| ``ID``        | Unique ID assigned to the job.                                               |
++---------------+------------------------------------------------------------------------------+
+| ``Type``      | Job type:                                                                    |
+|               |                                                                              |
+|               | -  ``TYPE_COMMAND``                                                          |
+|               | -  ``TYPE_EXPERIMENT``                                                       |
+|               | -  ``TYPE_NOTEBOOK``                                                         |
+|               | -  ``TYPE_SHELL``                                                            |
+|               | -  ``TYPE_TENSORBOARD``                                                      |
++---------------+------------------------------------------------------------------------------+
+| ``Job Name``  | Command or experiment name, also displayed in the WebUI.                     |
```

Comment:
I think we're going toward calling all of these jobs, at least internally. usage of "command" here could be confusing with command_type command